src/components/molecules/input-checkbox/ms-input-checkbox.ts

```diff
@@ -0,0 +1,35 @@
+import { LitElement, html } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+
+import { WithTheme } from '~/mixins'
+
+import { msInputCheckboxStyles } from './ms-input-checkbox.styles'
+
+@customElement('ms-input-checkbox')
+export class MSInputCheckbox extends WithTheme(LitElement) {
+  static styles = [msInputCheckboxStyles]
```

Comment:
```suggestion
  static styles = msInputCheckboxStyles
```
could you check if it works in this way?

Reply:
Yeah! its works! Done!